recipe/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "spacy-langdetect" %}
+{% set version = "0.1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Abhijit-2592/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  sha256: dc04e22e35f12d7fb4a86cda888da8d2529ca9569437cbd6925544babf1ee62e
+
+build:
+  # skipping s390x because of missing spacy
+  skip: True  # [s390x]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
+  number: 0
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools
+    - wheel
+    - spacy 3.7.2
+  run:
+    - python
+    - pytest
```

Comment:
```suggestion
```
It could be worth patching setup.py. pytest is not needed at runtime but spacy is.
https://github.com/Abhijit-2592/spacy-langdetect/blob/0.1.2/setup.py#L21

Reply:
will do